diff -Nru atat.orig/makefile atat/makefile
--- atat.orig/makefile	2021-06-02 16:42:24.132807000 +0100
+++ atat/makefile	2021-06-02 19:47:33.566818000 +0100
@@ -22,7 +22,7 @@
 	${MAKE} -C src clean
 
 install:
-	echo set atatdir="$(PWD)" > $(HOME)/.atat.rc
+	# echo set atatdir="$(PWD)" > $(HOME)/.atat.rc
 	${MAKE} -C src          BINDIR=${BINDIR} install
 	${MAKE} -C glue/jobctrl BINDIR=${BINDIR} install
 	${MAKE} -C glue/vasp    BINDIR=${BINDIR} install
