--- fsps/__init__.py.orig	2020-10-21 22:35:48.597032787 +0100
+++ fsps/__init__.py	2020-10-21 22:36:28.444406826 +0100
@@ -27,30 +27,6 @@
 except KeyError:
     raise ImportError("You need to have the SPS_HOME environment variable")
 
-# Check the githashes to make sure the required FSPS updates are
-# present, and if not or there are no githashes, raise an error
-REQUIRED_GITHASHES = ['6ad1058', 'b5250ab', 'a23e409', '45f9680','05584df']
-
-cmd = 'cd {0}; git log --format="format:%h"'.format(ev)
-stat, githashes, err = run_command(cmd)
-accepted = (len(githashes) > 0) and (len(err) == 0)
-if not accepted:
-    raise ImportError("Your FSPS version does not seem to be under git version "
-                      "control. FSPS is available on github at "
-                      "https://github.com/cconroy20/fsps and should be cloned from there")
-accepted = [req in githashes for req in REQUIRED_GITHASHES]
-if False in accepted:
-    reqs = ",".join([r[:-2] for r in REQUIRED_GITHASHES])
-    raise ImportError("Your FSPS version does not have correct history.  "
-                      "Perhaps you need to pull the most up-to-date FSPS version. "
-                      "Please make sure that you have the following commits "
-                      "in your git history, preferably with the last listed commit "
-                      "the same as your current FSPS commit githash: {0}".format(reqs))
-else:
-    # Store the githash.  If any version checking is going to happen,
-    # it should happen here
-    fsps_vers = githashes[0]
-
 # Only import the module if not run from the setup script.
 try:
     __FSPS_SETUP__
