--- libpyclingo/pyclingo.cc.orig	2020-11-19 10:37:13.872981703 +0000
+++ libpyclingo/pyclingo.cc	2020-11-19 10:37:25.825126362 +0000
@@ -1177,7 +1177,7 @@
     sizeof(T),                                  // tp_basicsize
     0,                                          // tp_itemsize
     PythonDetail::Get_tp_dealloc<T>::value,     // tp_dealloc
-    nullptr,                                    // tp_print
+    0,                                          // tp_print
     nullptr,                                    // tp_getattr
     nullptr,                                    // tp_setattr
     nullptr,                                    // tp_compare
