--- r4.0.6.orig/arch/arch-linux_gfortran.fcm	2021-06-07 19:22:46.947381000 +0100
+++ r4.0.6/arch/arch-linux_gfortran.fcm	2021-06-16 10:41:28.416858000 +0100
@@ -30,25 +30,26 @@
 #  - unix variables "$..." are accpeted and will be evaluated before calling fcm.
 #  - fcm variables are starting with a % (and not a $)
 #
-%NCDF_HOME           /usr/local/netcdf
-%HDF5_HOME           /usr/local/hdf5
-%XIOS_HOME           $HOME/xios-2.5
-%OASIS_HOME          /not/defined
+%NCDF_HOME           $EBROOTNETCDF
+%NCDFF_HOME          $EBROOTNETCDFMINFORTRAN
+%HDF5_HOME           $EBROOTHDF5
+%XIOS_HOME           $EBROOTXIOS
+%OASIS_HOME          $EBROOTOASIS3MINMCT
 
-%NCDF_INC            -I%NCDF_HOME/include -I%HDF5_HOME/include
-%NCDF_LIB            -L%NCDF_HOME/lib -lnetcdff -lnetcdf
+%NCDF_INC            -I%NCDF_HOME/include -I%NCDFF_HOME/include -I%HDF5_HOME/include
+%NCDF_LIB            -L%NCDF_HOME/lib -lnetcdf -L%NCDFF_HOME/lib -lnetcdff
 %XIOS_INC            -I%XIOS_HOME/inc 
-%XIOS_LIB            -L%XIOS_HOME/lib -lxios -L/usr/lib/gcc/x86_64-linux-gnu/5 -lstdc++
+%XIOS_LIB            -L%XIOS_HOME/lib -lxios -lstdc++
 
-%OASIS_INC           -I%OASIS_HOME/build/lib/mct -I%OASIS_HOME/build/lib/psmile.MPI1
+%OASIS_INC           -I%OASIS_HOME/include/mct -I%OASIS_HOME/include/psmile.MPI1
 %OASIS_LIB           -L%OASIS_HOME/lib -lpsmile.MPI1 -lmct -lmpeu -lscrip
 
 %CPP	             cpp -Dkey_nosignedzero 
-%FC	             /usr/bin/mpif90 -c -cpp 
-%FCFLAGS             -fdefault-real-8 -O3 -funroll-all-loops -fcray-pointer -ffree-line-length-none
+%FC	                 mpif90 -c -cpp 
+%FCFLAGS             -fdefault-real-8 -O3 -funroll-all-loops -fcray-pointer -ffree-line-length-none -fallow-argument-mismatch
 %FFLAGS              %FCFLAGS
 #%LD                  /usr/bin/mpif90 -Wl,-rpath=$HOME/INSTALL/lib:/usr/lib
-%LD                  /usr/bin/mpif90
+%LD                  mpif90
 %LDFLAGS             
 %FPPFLAGS            -P -C -traditional
 %AR                  ar
