--- hail/Makefile.orig	2021-01-26 15:17:37.588142000 +0000
+++ hail/Makefile	2021-01-26 15:23:25.321921136 +0000
@@ -17,6 +17,7 @@
 HAIL_PIP_VERSION := $(HAIL_MAJOR_MINOR_VERSION).$(HAIL_PATCH_VERSION)
 HAIL_VERSION := $(HAIL_PIP_VERSION)-$(SHORT_REVISION)
 ELASTIC_MAJOR_VERSION ?= 7
+INSTALLPATH = $(INSTALLPATH)
 
 
 $(eval $(call ENV_VAR,REVISION))
@@ -246,7 +247,7 @@
 .PHONY: install
 install: $(WHEEL)
 	-$(PIP) uninstall -y hail
-	$(PIP) install $(WHEEL)
+	$(PIP) install --prefix=$(INSTALLPATH) --verbose --no-deps --ignore-installed $(WHEEL)
 
 .PHONY: install-on-cluster
 install-on-cluster: $(WHEEL)
