diff -Nru scrappie-1.4.1.orig/build.py scrappie-1.4.1/build.py
--- scrappie-1.4.1.orig/build.py	2020-07-27 17:09:25.239622000 +0100
+++ scrappie-1.4.1/build.py	2020-07-27 17:11:45.165571000 +0100
@@ -8,7 +8,7 @@
     library_dirs=['/usr/local/lib/']
 else:
     # this might want to be cblas on some systems
-    libraries=['blas']
+    libraries=['openblas']
     library_dirs=[]
 
 if 'SCRAPPIESRC' in os.environ:
