--- setup.py.orig	2020-09-30 14:30:13.527363000 +0100
+++ setup.py	2020-09-30 14:49:28.841805000 +0100
@@ -4,6 +4,7 @@
 from torch.utils.cpp_extension import BuildExtension, CUDAExtension
 
 CUDA_FLAGS = []
+extra_compile_args = {"cxx": ["-O2"], "nvcc": ['-std=c++11', '-Xcompiler', '-mno-float128']}
 INSTALL_REQUIREMENTS = ['torch', 'numpy', 'scikit-image']
 
 def test_all():
@@ -15,15 +16,21 @@
     CUDAExtension('neural_renderer.cuda.load_textures', [
         'neural_renderer/cuda/load_textures_cuda.cpp',
         'neural_renderer/cuda/load_textures_cuda_kernel.cu',
-        ]),
+        ],
+        extra_compile_args=extra_compile_args,
+        ),
     CUDAExtension('neural_renderer.cuda.rasterize', [
         'neural_renderer/cuda/rasterize_cuda.cpp',
         'neural_renderer/cuda/rasterize_cuda_kernel.cu',
-        ]),
+        ],
+        extra_compile_args=extra_compile_args,
+        ),
     CUDAExtension('neural_renderer.cuda.create_texture_image', [
         'neural_renderer/cuda/create_texture_image_cuda.cpp',
         'neural_renderer/cuda/create_texture_image_cuda_kernel.cu',
-        ]),
+        ],
+        extra_compile_args=extra_compile_args,
+        ),
     ]
 
 setup(
